usage/rest-api/src/main/java/org/apache/brooklyn/rest/domain/CatalogItemSummary.java

```diff
@@ -121,6 +132,20 @@ public String getIconUrl() {
         return iconUrl;
     }
 
+    public Collection<Object> getTags() {
+        List<Object> result = new ArrayList<Object>();
+        for (Object t : tags) {
+            // TODO if we had access to a mapper we could use it to give better json
+            result.add(Jsonya.convertToJsonPrimitive(t));
+        }
+        return result;
+    }
+
+    @JsonIgnore
+    public Collection<Object> getRawTags() {
```

Comment:
Not directly in this project but in [brooklyn-tosca](https://github.com/cloudsoft/brooklyn-tosca/pull/30/files#diff-7bb08b08e5b7ef400557dbcb45441969R157). Although, it can be removed as per as my previous comment.